README.md

```diff
@@ -829,6 +856,29 @@ reference GitHub issues that this commit **Closes**.
 This is how a changelog based on this conventions is rendered:
 https://github.com/angular/angular/blob/master/CHANGELOG.md
 
+## Adding banners
+
+Adding banners on top of the production bundles is fully optional and disabled by default.
+
+It can be enabled with setting the `generateBanners` property within `baumeister.json` to `true`.
+
+```javascript
```

Comment:
Same as above and the comment may be redundant

Reply:
Used JavaScript Syntax to properly render the comments ;)